src/main/java/com/hedera/exchange/ERTNotificationHelper.java

```diff
@@ -90,4 +91,14 @@ public static void publishMessage(final String subject, final String message) {
 			LOGGER.error(Exchange.EXCHANGE_FILTER, "Failed to submit  {} : {} \n {}", subject, message, ex);
 		}
 	}
+
+	private static Regions getValidRegion(final String region) {
+		try {
+			return Regions.fromName(region);
+		} catch (IllegalArgumentException ex) {
+			LOGGER.warn(Exchange.EXCHANGE_FILTER, "Invalid region provided : {}, deafulting ot us-east-1",
```

Comment:
Should this be US_EAST_2?

Reply:
No we want to default to US_EAST_1 for testnet and mainNet lambdas